### DISCUSSION TITLE
Tree用checkStrictly做完全受控如何做到父节点半选中样式

### DISCUSSION THREAD

**Discussion by ortizyc** [ant-design/ant-design#32055] [Q&A]
Tree用checkStrictly做完全受控如何做到父节点半选中样式

**Comment by yoyo837** [comment]
参照文档就可以了, 用了checkStrictly 表示父子脱离关联关系, 你给数据给组件, 想哪个选中就哪个选中

**Reply by ortizyc** [reply]
使用了checkStrictly后，没有办法像未使用一样展示出半选中状态了，希望可以加入半选中的特性
